Zype/Helpers/AnalyticsProtocolDelegate.swift

```diff
@@ -60,4 +61,77 @@ extension PlayerVC: AnalyticsHelperProtocol {
         
         return (beacon: configUrl, customDimensions: customDimensions)
     }
+
+    func getSegmentAnalyticsFromResponse(_ playerObject: VideoObjectModel?) -> [String: Any] {
+        var customDimensions = [String: Any]()
+        customDimensions[SegmentAnalyticsAttributes.contentCmsCategory.rawValue] = "null"
+        customDimensions[SegmentAnalyticsAttributes.adType.rawValue] = "null"
+        customDimensions[SegmentAnalyticsAttributes.contentShownOnPlatform.rawValue] = "ott"
+        customDimensions[SegmentAnalyticsAttributes.streaming_device.rawValue] = "Apple AppleTV"
+        customDimensions[SegmentAnalyticsAttributes.videoAccountId.rawValue] = "416418724"
+        customDimensions[SegmentAnalyticsAttributes.videoAccountName.rawValue] = "People"
```

Comment:
This value should be pulled from the App Title, not hardcoded

Reply:
😞 got confused after looking at the APPS-784 description that it's hardcoded, that's fine for PeopleTv but this is base repo so it should not be hardcoded. Thanks for catching this in review.